### PR TITLE
Improvements to Notification examples

### PR DESCRIPTION
**Pull request by mstahv**

### CODE REVIEW

File: src/main/java/com/vaadin/demo/component/notification/NotificationUndo.java [hunk 1]
```diff
@@ -66,6 +55,25 @@ public Notification show() {
         return notification;
     }
 
+    public class UndoButton extends Button {
+        public UndoButton() {
+            super("Undo");
+            addThemeVariants(ButtonVariant.LUMO_TERTIARY_INLINE);
+            setAriaLabel("Close");
```
Comment (OlliTietavainenVaadin) [inline src/main/java/com/vaadin/demo/component/notification/NotificationUndo.java:62]: Should this aria label say "Undo" instead of "Close"?

Comment (mstahv) [reply src/main/java/com/vaadin/demo/component/notification/NotificationUndo.java:62]: The original code (before refactoring) had no ARIA label. Maybe I'll remove that altogether, as I think all those ARIA label's are irrelevant for these examples and should be (and maybe are) covered in some other part of our docs.

Comment (mstahv) [reply src/main/java/com/vaadin/demo/component/notification/NotificationUndo.java]: removed

**Review by jouni — APPROVED** [review]

**Comment by jouni** [comment]
Most of the inline styling could be replaced with Lumo utility classes, if you wish to clean it up a bit more.

**Pull request merged**